examples/temperature/temperature-service/configs/temperature-service.json

```diff
@@ -0,0 +1,86 @@
+{
```

Comment:
Are there no license for the service manifests?
At least there should be copyright.